doc/sphinx/source/community/backward_compatibility.rst

```diff
@@ -0,0 +1,262 @@
+ESMValTool policy on backward compatibility
+===========================================
+
+V1.0. Agreed with ESMValTool community during October 2022 workshop.
+
+Motivation
+----------
+
+Development of recipes or conducting project-related work may require a
+rather long period of time during which new versions of the ESMValTool
+might become available. For a good user experience and a seamless
+workflow, users and developers need to know before upgrading to a new
+version if and how their work might be affected (backward
+compatibility). This includes, for instance, information about changes
+to technical features such as syntax of recipes and configuration files,
+and interfaces of shared functions, but also changes that affect the
+results of an ESMValTool run, e.g. modification of algorithms or changes
+to the order of operators. It is therefore essential that users and
+developers have the best advice on how and when to upgrade to new
+versions.
+
+While trying to minimise the impact of new code developments on users
+and developers by maintaining backward compatibility where possible,
+this cannot always be guaranteed. A very restrictive policy might delay
+the ESMValTool development and make it more complex for developers to
+contribute.
+
+This document outlines the key principles and proposals of an updated
+ESMValTool policy on backward compatbility.
+
```

Comment:
```suggestion
This document outlines the key principles and proposals of an updated
ESMValTool policy on backward compatbility.

```
Once merged, this will be the policy, not a proposal

Reply:
Change "compatbility" to "compatibility".

Reply:
Agreed. "Proposals" now removed and spelling corrected.